tests/StoringGroceries.tex

```diff
@@ -129,6 +129,10 @@ \subsection{OC instructions}
 \textbf{2 hours before the test}
 \begin{itemize}
     \item Announce the startup location for robots.
+    % This test is about manipulation and object recognition, NOT about finding furniture
+    % Finding a table is done in the Restaurant task
+    \item Announce which table will be used in the test
+    \item Announce a rough location for the table, i.e. left or right of the robot's startup location
```

Comment:
Would change it to:
Announce where the table will be placed (e.g. left or right).